docs/dictionary/message/resizeControl.lcdoc

```diff
@@ -9,6 +9,8 @@ Sent to a <control(glossary)> right after the user resizes it.
 
 Introduced: 1.0
 
+Associations: field, button, graphic, scrollbar, player, image
```

Comment:
It should, I will file a bug report. For the purposes of the default handler stuff, widgets don't list handlers with widget association yet so I'll fix both at the same time